app/controllers/api/v2/certificates_controller.rb

```diff
@@ -10,11 +10,6 @@ def search
       end
 
       def index
-        certificates = Certificate.actual
-                                  .eager(:certificate_descriptions, :certificate_description_periods, :certificate_type_description)
-                                  .order(Sequel.asc(%i[certificate_type_code certificate_code]))
-                                  .all
-
         render json: Api::V2::Certificates::CertificateListSerializer.new(certificates).serializable_hash
```

Comment:
We could safely share this serializer with the search action as long as the certificate type description was being added in the cache serializer